### PR TITLE
[db] Remove user storage db model

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Looks to be unused, no API calls into it in the last 3 months.
Anton provided context this was used before current vs code sync

## Related Issue(s)

Fixes #

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - mp-db-remob27d2ae0f6</li>
	<li><b>🔗 URL</b> - <a href="https://mp-db-remob27d2ae0f6.preview.gitpod-dev.com/workspaces" target="_blank">mp-db-remob27d2ae0f6.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

**Review by geropl — APPROVED** [review]

File: components/server/src/code-sync/code-sync-service.ts [hunk 1]
```diff
@@ -365,58 +296,34 @@ export class CodeSyncService {
         }
 
         let content: string;
-        if (resourceRev === fromTheiaRev) {
```
Comment (akosyakov) [inline components/server/src/code-sync/code-sync-service.ts:368]: there are still some references to fromTheiaRev above, but not sure about semantic, there are should be unit tests though

**Review by filiptronicek — COMMENTED** [review]
Looks like settings sync is broken 🧐 

image

**Comment by easyCZ** [comment]
Thanks @filiptronicek & @akosyakov . My attempt at the removal was pretty crude. Can you see a way (from either eng, or product) side to remove the old resource or we still need to keep it?

**Comment by easyCZ** [comment]
Closing as in the current state this is not mergeable. I'll leave it with Team IDE to follow-up on how they want to move forward, or if they want to deprecate support for old plugins.

**Pull request closed without merging**